src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaServiceTest.java

```diff
@@ -338,12 +339,51 @@ void assigneeTask_should_throw_resource_not_found_exception_when_other_exception
             );
 
         doThrow(exception)
-            .when(camundaServiceApi).assigneeTask(eq(taskId), anyMap());
+            .when(camundaServiceApi).assignTask(eq(taskId), anyMap());
 
-        assertThatThrownBy(() -> camundaService.assigneeTask(taskId, userId))
+        assertThatThrownBy(() -> camundaService.assignTask(taskId, userId))
+            .isInstanceOf(ServerErrorException.class)
+            .hasCauseInstanceOf(FeignException.class);
+
+    }
+
+    @Test
+    void assigneeTask_should_throw_decoded_exception_when_other_exception_is_thrown() {
```

Comment:
should this be **assign**Task_should_throw_decoded_exception_when_other_exception_is_thrown and not **assignee**Task_should_throw_decoded_exception_when_other_exception_is_thrown?